SR2025/2025-01-15-check-in.md

```diff
@@ -0,0 +1,27 @@
+---
+to: SR2025 teams
+subject: Student Robotics 2025 {TLA} January Check-in
+---
+
+Hi {PrimaryGivenName},
+
+Happy New Year!
+
+Just wanted to check in on how your team is doing?
+
+It would be great to hear what your team is currently working on and if there's
+anything you'd like a hand with.
+
+A couple of quick reminders for the next few weeks:
+
+* if you'd like to come to the [Cambridge Tech Day][cambridge-tech-day] on
```

Comment:
Suggestion: Given the number of sign-ups we've had, it's probably worth including more details on the tech day, get more people engaged.

Reply:
Agree we could do more to illicit sign-ups, however this email is not that. (See discussion around the subject line).

Reply:
Given the focus of this email being the check-in, the other parts are deliberately fairly brief. In that context, it is completely reasonable to omit further details. (Details and brevity are usually opposing forces, we need to draw a line somewhere. For what this email is, it is a deliberate choice to move that line in favour of brevity. We can and will send other emails with more details about the Tech Days).